buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchCluster.java

```diff
@@ -295,6 +295,27 @@ private void commonNodeConfig() {
         }
     }
 
+    public void run(Boolean debug) {
```

Comment:
Why don't we just have `RunTask` extend `DefaultTestClustersTask` and let the test clusters plugin handle the lifecycle stuff here?